lib/fencing/st_client.c

```diff
@@ -442,17 +442,22 @@ make_args(const char *action, const char *victim, uint32_t victim_nodeid, GHashT
             alias = g_hash_table_lookup(port_map, victim);
         }
 
-        /* Always supply the node's name too:
-         *    https://fedorahosted.org/cluster/wiki/FenceAgentAPI
-         */
-        append_const_arg("nodename", victim, &arg_list);
```

Comment:
Even if the stonith agent checks "port" for validation, an user could use a number as the host name... 

Although I was thinking "nodename" and "nodeid" actually also belong to parameters reserved by fence-agents, but I think we can stick with https://github.com/ClusterLabs/pacemaker/commit/6497493, unless anyone else complains.